.github/workflows/ci.yaml

```diff
@@ -221,3 +233,77 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./doc
+
+  obs-commit:
+    name: Commit the project on OBS
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' || (github.event_name == 'push' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch'
+    needs: [static-code-analysis, test]
+    container:
+      image: ghcr.io/trento-project/continuous-delivery:main
+      env:
+        GITHUB_OAUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        DEST_FOLDER: "/tmp/osc_project"
+        NAME: trento-wanda-image
```

Comment:
This naming could be confusing, as the container in the public environment is named `wanda`, without any trento prefix.
We should think carefully how we want to name this

Reply:
I see. It just follows the same naming pattern we use for web.
| Project | gh image name | OBS/IBS image name |
|---	|---	|---	|
| web | `trento-web` | `trento-web-image` |
| wanda | `wanda` | `trento-wanda-image` |

We are somewhat consistently inconsistent :sweat_smile: 
What should we do?

Reply:
Options:
- rename the gh image to `trento-wanda`.
- move away from `wanda` to something checks engine related (`trento-checks`, `trento-checks-engine` or similar)

I'd go for the first one, sounds simpler.

Reply:
+1 to keep consitency with the trento prefix if possible

Reply:
https://github.com/trento-project/wanda/pull/173/commits/8a1590b339ad85aaee969d809d395871535e8d85